Example Code/Reflowduino_Demo/Reflowduino_Demo.ino

```diff
@@ -205,31 +260,103 @@ void loop() {
         t_final = (T_preheat - T_start) / preheat_rate + t_start;
         // Calculate desired temperature at that instant in time using linear interpolation
         setPoint = duration * (T_preheat - T_start) / (t_final - t_start);
+        if (false && printThisLoop) {
```

Comment:
I think it might be better to use something like #ifdef DEBUG at the top of the code to disable/enable all of these lines with a single line.

Reply:
yeah, I thought setpoint and final were going to be more readable than they are :)